scipy/stats/_fft_char_fn.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
```

Comment:
This file is kind of odd to have separate, and it's not clear what it's for when you open it. Given that it's only used from one other file and is only 76 lines long, can the contents just be included in `_levy_stable_distn.py`?